code/modules/mob/living/carbon/xenomorph/castes/runner/abilities_runner.dm

```diff
@@ -112,7 +113,20 @@
 	action_icon_state = "evasion_[auto_evasion? "on" : "off"]"
 	update_button_icon()
 
-/datum/action/ability/xeno_action/evasion/action_activate()
+/datum/action/ability/xeno_action/evasion/action_activate(deactivate = TRUE)
```

Comment:
This is smelly, Don't do this.

Just make the proc itself check if it's active, and deactive if it is instead of activate.